Makefile

```diff
@@ -789,6 +795,18 @@ keystone_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_de
 	make keystone_cleanup
 	make mariadb_cleanup
 
+.PHONY: octavia_kuttl_run
+octavia_kuttl_run: ## runs kuttl tests for the octavia operator, assumes that everything needed for running the test was deployed beforehand.
+	INSTALL_YAMLS=${INSTALL_YAMLS} kubectl-kuttl test --config ${OCTAVIA_KUTTL_CONF} ${OCTAVIA_KUTTL_DIR}
+
+.PHONY: octavia_kuttl
+octavia_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy keystone octavia keystone_deploy mariadb_deploy_validate ## runs kuttl tests for the octavia operator. Installs openstack crds and mariadb, keystone, octavia operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
```

Comment:
`octavia_deploy` should be added as well, shouldn't it?

Reply:
Not really, the deployment is done as part of the tests